haystack/nodes/file_converter/markdown.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import re
+import frontmatter
```

Comment:
Could you put the `import frontmatter` in the `try:` block on line 7? Since this will be an optional package that can be installed using the `preprocessing` dependencies we want to catch the `ImportError` and print the optional component not installed.